src/main/java/com/book/booksstore/controller/BookController.java

```diff
@@ -24,7 +24,7 @@
 @Tag(name = "Book store", description = "Endpoints for managing books.")
 @RequiredArgsConstructor
 @RestController
-@RequestMapping("/books")
+@RequestMapping("/api/books")
```

Comment:
add `server.servlet.context-path=/api` to application.properties
```suggestion
@RequestMapping("/books")
```

Reply:
done